packages/central-server/__tests__/hl7fhir/materialised/Patient.test.js

```diff
@@ -58,7 +65,7 @@ describe(`Materialised FHIR - Patient`, () => {
         address: [
           {
             city: additionalData.cityTown,
-            line: [additionalData.streetVillage],
+            line: [village.name],
```

Comment:
Do we want a test that it maintains the fallback? Maybe that's implicit in some other test in this file, I haven't checked

Reply:
Yeah that functionality is tested in most of the other tests in this suite. Good point though, I was a bit lazy not creating an explicit test for this behaviour. But not too fussed as plan to change it rather soon.